src/fundus/parser/base_parser.py

```diff
@@ -70,9 +70,9 @@ def __lt__(self, other):
 
     def __repr__(self):
         if instance := self.__self__:
-            return f"bound {type(self).__name__} of {instance}: {self.__wrapped__} --> '{self.__name__}'"
+            return f"bound {type(self).__name__} of {instance}: {self.__wrapped__} --> !r{self.__name__}"
```

Comment:
Unfortunately, it seems that you have misused the `!r` operator. It's supposed to be placed inside the curly braces like this:
```python
... {self.__name__!r}
```

Reply:
Has been done.